samples/excel/85-preview-apis/culture-info-date-time.yaml

```diff
@@ -0,0 +1,138 @@
+order: 5
+id: excel-culture-info-date-time
+name: 'Culture info: date and time'
+description: This sample shows how to apply the cultural settings APIs to retrieve read-only operating system date and time settings.
+host: EXCEL
+api_set:
+    ExcelAPI: '1.12'
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#display-date-time-setting").click(() => tryCatch(displayDateTimeSetting));
+        $("#write-date-time-setting").click(() => tryCatch(writeDateTimeSetting));
+
+        /* Create table with only header content. */
+        async function setup() {
+          await Excel.run(async (context) => {
+            context.workbook.worksheets.getItemOrNullObject("Sample").delete();
+            const sheet = context.workbook.worksheets.add("Sample");
+            const range = sheet.getRange("A1:B6");
+            range.values = [["Culture Setting", "Setting Format"], ["", ""], ["", ""], ["", ""], ["", ""], ["", ""]];
+            const table = sheet.tables.add("A1:B6", true);
+            range.format.autofitColumns();
+            sheet.activate();
+            await context.sync();
+          });
+        }
+
+        /* Use the cultural settings API to display date and time settings in your console. */
+        async function displayDateTimeSetting() {
+          await Excel.run(async (context) => {
+            context.application.cultureInfo.datetimeFormat.load([
```

Comment:
Consider adding a comment like "Load the date and time settings from the workbook."

Reply:
Since these are system settings (based on all the info I have), would you agree with "Load the system date and time settings." as a comment instead?

Reply:
Yeah, that works!